app/controllers/courses_controller.rb

```diff
@@ -24,6 +24,15 @@ def destroy
     head :no_content
   end
 
+  # get visible returns courses that are taught by an admin or enrolled in by a student
+  def get_visible
```

Comment:
Hmm, we don't want to return everything for the user call. Like we're not also going to return all the assignments and problems. So I suppose we could move this to users, but I'm not totally convinced that it belongs there and I think it makes sense to have a separate call.